ee/clickhouse/views/test/test_clickhouse_trends.py

```diff
@@ -62,6 +62,41 @@ def test_insight_trends_basic(self):
             [str(created_people["1"].uuid), str(created_people["2"].uuid)]
         )
 
+    @snapshot_clickhouse_queries
+    def test_insight_trends_clean_arg(self):
+
+        events_by_actor = {
+            "1": [{"event": "$pageview", "timestamp": datetime(2012, 1, 14, 3), "properties": {"key": "val"}},],
+            "2": [{"event": "$pageview", "timestamp": datetime(2012, 1, 14, 3)},],
+        }
+        created_actors = journeys_for(events_by_actor, self.team)
+
+        with freeze_time("2012-01-15T04:01:34.000Z"):
+
+            request = TrendsRequest(
+                date_from="-14d",
+                display="ActionsLineGraph",
+                events=[
+                    {
+                        "id": "$pageview",
+                        "math": None,  # this argument will now be removed from the request instead of becoming a string
```

Comment:
Is it possible for the deserialization to validate this as `None` instead of `"None"`. This was we can ensure that at the boundaries of the process we are enforcing a specific structure and thus reduce the number of cases we have to consider throughout the app. I don't know if that's possible, I haven't looked at the deserialization.

Reply:
```
data = {
      key: None if value == 'None' else value
      for key, value in data.items()
}
```
could do this in the filter/basefilter init